README.md

```diff
@@ -75,7 +75,7 @@ By default all votes are positive, so `@user3` has cast a 'good' vote for `@post
 
 `@user1`, `@user3`, and `@user5` all voted in favor of `@post`.
 
-`@user2` and `@user4` on the other had has voted against `@post`.
+`@user2` and `@user4` voted against `@post`.

```

Comment:
Counter propsal :)
Maybe this is event more succinct, clearer and to the point @ryanto

Reply:
Love it! Thanks again for getting this fixed